examples/destructure-tuples/destructure_tuples.md

```diff
@@ -0,0 +1,7 @@
+# Destructure tuples
+
+Tuples can be accessed normally like arrays by specifying the index. However, there is a more intuitive way to use tuples by destructuring them. With this approach, you can read tuple values in a more idiomatic way within the code instead of using index-based access.
```

Comment:
```suggestion
Tuples can be accessed normally like arrays by specifying the index. However, there is a more intuitive way to use tuples by destructuring them. With this approach, you can read tuple values in a more idiomatic way within the code instead of using index-based (member) access.
```